tests/projects/external-kit/Twoliter.toml

```diff
@@ -0,0 +1,19 @@
+schema-version = 1
+release-version = "1.0.0"
+
+[sdk]
+name = "bottlerocket-sdk"
+vendor = "bottlerocket"
+version = "0.41.0"
+
+[vendor.bottlerocket]
+registry = "public.ecr.aws/bottlerocket"
+
+[vendor.custom-vendor]
+# We need to figure out how we can do integration testing with this
+registry = "public.ecr.aws/bottlerocket"
```

Comment:
This is fine for now, but we will want to have some way to run external-kit tests and we won't be pushing these artifacts to our production ECR domain.